### PR TITLE
Redis name configurable in terraform

### PR DESCRIPTION
**Pull request by chaodaiG**
/assign @sethvargo

### CODE REVIEW

File: terraform-e2e/README.md [hunk 1]
```diff
@@ -26,23 +28,6 @@ terraform import module.en.google_app_engine_application.app ${PROJECT_ID}
...
-
-  on ../terraform/main.tf line 15:
-  15: provider "google" {
-
-The configuration for
-module.en.provider["registry.terraform.io/hashicorp/google"] depends on values
-that cannot be determined until apply.
-```
-
-This can be mitigated by commenting out [project = var.project](https://github.com/google/exposure-notifications-verification-server/blob/86a05f09dabfeddbce73c9d628ec01d3738782ed/terraform/main.tf#L16), as well as [project = var.project](https://github.com/google/exposure-notifications-verification-server/blob/86a05f09dabfeddbce73c9d628ec01d3738782ed/terraform/main.tf#L23). This is similar to the issue described in https://github.com/hashicorp/terraform/issues/25816, and might be fixed in 0.13.1 release of terraform.
```
Comment (chaodaiG) [inline terraform-e2e/README.md:44]: This had been confirmed fixed in terraform 1.13.1

File: terraform-e2e/README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ This directory contains terraform configuration to be used for deploying verific
 
 ## Prerequisite
 
+Terraform version > 1.13.1
```
Comment (sethvargo) [inline terraform-e2e/README.md:7]: We don't need to document this. If 1.13 is a requirement, we should add this to the terraform configs under `required_version` (we already have 1.13 there, just bump it). Better to capture this as code that docs.

File: terraform-e2e/README.md (hunk 1 above)
Comment (sethvargo) [reply terraform-e2e/README.md:44]: I'd rather leave this in with a note at the top saying "upgrade to the latest Terraform version first"

File: terraform-e2e/main.tf [hunk 3]
```diff
@@ -28,6 +28,7 @@ module "en" {
 
   project = var.project
   database_name = "en-verification-${random_string.suffix.result}"
+  redis_name = "verification-cache-${random_string.suffix.result}"
```
Comment (sethvargo) [inline terraform-e2e/main.tf:31]: Please run `terraform fmt` or install an editor plugin that does auto-formatting.

File: terraform/variables.tf [hunk 4]
```diff
@@ -66,6 +66,11 @@ variable "storage_location" {
   default = "US"
 }
 
+variable "redis_name" {
+  type = string
```
Comment (sethvargo) [inline terraform/variables.tf:70]: Please run `terraform fmt`.

File: terraform-e2e/README.md (hunk 1 above)
Comment (chaodaiG) [reply terraform-e2e/README.md:44]: Done

File: terraform-e2e/README.md (hunk 2 above)
Comment (chaodaiG) [reply terraform-e2e/README.md]: Added in code

File: terraform-e2e/main.tf (hunk 3 above)
Comment (chaodaiG) [reply terraform-e2e/main.tf]: done

File: terraform/variables.tf (hunk 4 above)
Comment (chaodaiG) [reply terraform/variables.tf]: done

**Pull request merged**